content/04.workflows-and-software.md

```diff
@@ -36,22 +36,21 @@ Here, we draw a distinction between two types of workflows: "research" workflows
 
 **Choosing a workflow system**
 If your use case extends beyond these tools, there are several scriptable workflow systems that offer comparable benefits for carrying out your own data-intensive analyses.
 Each has it own strengths, meaning each workflow software will meet an individuals computing goals differently (see **Table @tbl:workflows**).
-Our lab has adopted Snakemake, in part due to its similarity and integration with Python, its flexibility for building and testing new analyses in different languages, and its intuitive integration with software management tools (described below)[@doi:10.1093/bioinformatics/bts480].
-Snakemake and Nextflow are commonly used for devloping new research pipelines, where flexibility and iterative, branching development is a key feature [@doi:10.1038/nbt.3820]. Common Workflow Language (CWL) and Workflow Description Language (WDL) are workflow specification formats that are more geared towards scalability, making them ideal for production-level pipelines with hundreds of thousands of samples [@doi:10.6084/m9.figshare.3115156.v2]. WDL and CWL are commonly executed on platforms such as Terra [@url:https://terra.bio] or Seven Bridges Platform [@url:https://www.sevenbridges.com/platform/].
-Language-specific workflow systems, such as ROpenSci's Drake [@doi:10.21105/joss.00550], are limited in the scope of tasks they can execute, but are powerful within their language and easier to integrate for those comfortable with that language.
+Our lab has adopted Snakemake [@doi:10.1093/bioinformatics/bts480 @doi:10.5281/zenodo.4067137], in part due to its similarity and integration with Python, its flexibility for building and testing new analyses in different languages, and its intuitive integration with software management tools (described below).
+Snakemake and Nextflow [@doi:10.1038/nbt.3820] are commonly used for developing new research pipelines, where flexibility and iterative, branching development is a key feature. Common Workflow Language (CWL) and Workflow Description Language (WDL) are workflow specification formats that are more geared towards scalability, making them ideal for production-level pipelines with hundreds of thousands of samples [@doi:10.6084/m9.figshare.3115156.v2]. WDL and CWL are commonly executed on platforms such as Terra [@url:https://terra.bio] or Seven Bridges Platform [@url:https://www.sevenbridges.com/platform/].
+Language-specific workflow systems, such as ROpenSci's Drake [@doi:10.21105/joss.00550], are more limited in the scope of tasks they can execute, but are powerful within their language and easier to integrate for those comfortable with that language.
```

Comment:
instead of "are powerful within their language" say "can take full advantage of the language's internal data structures"? (This is a soft suggestion, doesn't quite capture it :)